[2025-07-07T07:48:33.182+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: ETL_airflare_data_test.extract manual__2025-07-07T07:48:32.044989+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
